packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginFormError.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Collapse from 'react-collapse';
+import { default as cx } from 'classnames';
 
-const LoginFormError = ({ children, show, className, ...props }) => (
-  <div className={`login-form-error ${className}`}>
-    <Collapse {...props} isOpened={show}>
+const LoginFormError = ({ children, show, className, ...props }) =>
+  children && (
+    <Collapse {...props} isOpened={show} className={cx('login-form-error', className)}>
```

Comment:
Should probably exclude `className` from the props just so it isn't passed twice if specified.

```suggestion
    <Collapse {...excludeKeys(props, ['className'])} isOpened={show} className={cx('login-form-error', className)}>
```